### ISSUE TITLE
Enderchest inventories clearing out at random times.

### ISSUE THREAD

**Issue by gabizou** [Multiverse/Multiverse-Inventories#84]
So I've updated to MV-Inv 2.5 build 195, dumptruckman requested I include this link http://j.mp/PWNBbq for knowing what I'm running. For one let me explain what I have so far:

I have OpenInv and Multiverse-Inventories. I know that OpenInv should be opening the inventory of anything and it was compatible with MV-Inv before 1.3. Now with enderchests having no API (I understood that since the beginning), MV-Inventories had to come up with a way to handle the API-less enderchests. I know that it would wipe all the enderchest inventories and all inventories were saved in the MV-Inv format. Since then, enderchests worked fine.

However until a few days ago, people have been saying that their enderchest inventories are now empty. I verified with OpenInv (technically /openender <player>) and they are indeed empty. Any edits done with OpenInv results in no edits and the items aren't put through. I believe this is because OpenInv is attempting to read the player.dat inventory vs MV-Inventory's inventory. However,  my backups from even last week still show empty enderchest inventory for all players minus OP's.

TLDR: Enderchests worked until yesterday where people were reporting their items are gone from their enderchests. I haven't changed anything to the server or the plugins in recent days (tbh I've been testing on another server folder so my players don't get any sort of issue from tests).

**Comment by dumptruckman** [comment]
This ticket is old and may no longer be relevant. I apologize for not being able to help with it in a timely manner. I am going to close it now but if you feel the issue is still relevant please open a new ticket and I may be able to assist with it this time around.
